File: Uni.py
#Ingreso de datos
import string


prom = float(input("Ingrese el promedio del estudiante: "))
est = input("Ingrese si es de pregrado o posgrado: ")

#Calculos
if prom >= 4.5 and est == "pregrado":
    cre = 28
    des = cre * 0.25
    tot = (cre * 50000) - des
    print("El estudiante debe pagar ", tot, "pesos por " , cre, "creditos" )

elif prom >= 4.0 and prom < 4.5 and est == "pregrado":
    cre = 25
    des = cre * 0.1
    tot = (cre * 50000) - des
    print("El estudiante debe pagar ", tot, "pesos por " , cre, "creditos" )

elif prom > 3.5 and prom < 4.0 and est == "pregrado":
    cre = 20
    tot = cre * 50000
    print("El estudiante debe pagar ", tot, "pesos por " , cre, "creditos" )

elif prom >= 2.5 and prom < 3.5 and est == "pregrado":
    cre = 15
    tot = cre * 50000
    print("El estudiante debe pagar ", tot, "pesos por " , cre, "creditos" )

elif prom < 2.5 and est == "pregrado":
    cre = 0
    tot = cre * 50000
    print("El estudiante debe pagar ", tot, "pesos por " , cre, "creditos" )

elif prom >= 4.5 and est == "posgrado":
    cre = 20
    des = cre * 0.2
    tot = (cre * 300000) - des
    print("El estudiante debe pagar ", tot, "pesos por " , cre, "creditos"  )

elif prom < 4.5 and est == "posgrado":
    cre = 10
    tot = cre * 300000
    print("El estudiante no tiene derecho a matricula")

File: Uni.cpp
#include <iostream>
using namespace std;

int main()
{
   int vCreditos, nCreditos;
   float Prom, pDescuento;
   char g;
   cout << "Ingrese su promedio: ";
   cin >> Prom;
  
   cout << "Ingresa el NUMERO  si es pregrado(1) o posgrado(2): ";
   cin >> g;
   if (g == 1 && Prom >= 4.5){
       nCreditos = 28;
       pDescuento = 0.25;
       cout << "El monto a pagar es de: " << (nCreditos * 50000) - ((nCreditos * 50000) * pDescuento);
   }
   else if (g == 1 && Prom < 4.5 && Prom >= 4.0){
       nCreditos = 25;
       pDescuento = 0.1;
       cout << "El monto a pagar es de: " << (nCreditos * 50000) - ((nCreditos * 50000) * pDescuento);
   }
   else if (g == 1 && Prom < 4.0 && Prom >= 3.5){
       nCreditos = 20;
       pDescuento = 0;
       cout << "El monto a pagar es de: " << (nCreditos * 50000) - ((nCreditos * 50000) * pDescuento);
   }
   else if (g == 1 && Prom < 3.5 && Prom >= 2.5){
       nCreditos = 15;
       pDescuento = 0;
       cout << "El monto a pagar es de: " << (nCreditos * 50000) - ((nCreditos * 50000) * pDescuento);
   }
   else if (g == 1 && Prom < 2.5){
       cout << "No se puede matricular";
   }
   else if (g == 2 && Prom >= 4.5){
       nCreditos = 20;
       pDescuento = 0.2;
       cout << "El monto a pagar es de: " << (nCreditos * 300000) - ((nCreditos * 300000) * pDescuento);
   }
   else if (g == 2 && Prom < 4.5){
       nCreditos = 10;
       pDescuento = 0;
       cout << "El monto a pagar es de: " << (nCreditos * 50000) - ((nCreditos * 50000) * pDescuento);
   }
}